### PR TITLE
Fix/range order arbitrum integration

### PR DESCRIPTION
**Pull request by thegeronimo**
# Task:

## Description

Added integration tests and fixes against arbitrum alpha deployment

## Type of change

- [X ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Document update
- [ ] Quality of life improvement
- [ ] Package upgrades

## Checklist

- [ X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I have NATSPEC'd any new functions
- [ X] If appropriate, I have properly tested my new functionality

### CODE REVIEW

File: packages/contracts/contracts/utils/UniswapConversionsTest.sol [hunk 1]
```diff
@@ -21,6 +21,15 @@ contract UniswapConversionsTest {
         return sqrtPriceX96ToUint(sqrtPriceX96, token0Decimals);
     }
 
+    function priceToSqrt(uint256 weiPrice, bool inversed, uint8 token0Decimals)
+        public
+        pure
+        returns (uint160 sqrtPriceX96)
+    {
+        uint256 token0Token1 = inversed ? 10 ** token0Decimals / weiPrice : weiPrice;
```
Comment (kjr217) [inline packages/contracts/contracts/utils/UniswapConversionsTest.sol:29]: we comfortable with the truncation here?

File: packages/contracts/contracts/hedging/UniswapV3RangeOrderReactor.sol [hunk 2]
```diff
@@ -504,15 +500,19 @@ contract UniswapV3RangeOrderReactor is IUniswapV3MintCallback, IHedgingReactor,
...
      * @return params the parameters needed to mint a range order with average price when filled
      */
     function _getTicksAndMeanPriceFromWei(uint256 price, RangeOrderDirection direction, bool inversed)
         private
         view
         returns (RangeOrderParams memory) {
-        uint160 sqrtPriceX96 = _sqrtPriceFromWei(price);
+        uint160 sqrtPriceX96;
+        {
+            uint256 convertedPrice = OptionsCompute.convertToDecimals(price, token1.decimals());
```
Comment (kjr217) [inline packages/contracts/contracts/hedging/UniswapV3RangeOrderReactor.sol:513]: why are these hardcoded to token0 and token1?

Comment (thegeronimo) [reply packages/contracts/contracts/hedging/UniswapV3RangeOrderReactor.sol:513]: That's the format Uniswap uses. They are not hardcode, they are set when the contract is initialized. `sqrtPriceFromWei` takes a price converts to sqrtPriceX96 token0/token1 format, it needs token0 decimals to do so.

File: packages/contracts/contracts/utils/UniswapConversionsTest.sol (hunk 1 above)
Comment (thegeronimo) [reply packages/contracts/contracts/utils/UniswapConversionsTest.sol:29]: Yes, if something is originally 6 decimals then converted to 18 it's been padded so truncation gets it back to the original state.

**Review by kjr217 — APPROVED** [review]

**Pull request merged**